pkg/controller/trustedcabundle/controller.go

```diff
@@ -64,23 +64,21 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 
 var _ reconcile.Reconciler = &ReconcileTrustedCABundle{}
 
-// ReconcileProxyConfig reconciles a ClusterLogging object
+//ReconcileTrustedCABundle reconciles the kibana resource on
+//creates or update events of the trusted CA bundle config map.
 type ReconcileTrustedCABundle struct {
 	// This client, initialized using mgr.Client() above, is a split client
 	// that reads objects from the cache and writes to the apiserver
 	client client.Client
 	scheme *runtime.Scheme
 }
 
-// Reconcile reads that state of the trusted CA bundle configmap objects for the
-// collector and the visualization resources.
+// Reconcile reads that state of the trusted CA bundle configmap objects for Kibana resource.
 // When the user configured and/or system certs are updated, the pods are triggered to restart.
 func (r *ReconcileTrustedCABundle) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	if request.Name == constants.KibanaTrustedCAName {
-		if err := kibanahandler.ReconcileKibanaInstance(request, r.client); err != nil {
-			// Failed to reconcile - requeuing.
-			return reconcileResult, err
-		}
+	if err := kibana.ReconcileKibanaInstance(request, r.client); err != nil {
```

Comment:
so the reason we had the check before was because we only wanted to reconcile Kibana when our expected trustbundle was updated.... if this is causing it to not go through then something else may be the issue...

Reply:
I believe this was an obsolete double check, because already handle this checks in https://github.com/openshift/elasticsearch-operator/pull/351/files#diff-d77a5e8d053611814893162d4e4d0b18R52

Reply:
yup, i realized this after looking at your CLO change. my mistake :)